Documentation/Upgrade/rook-upgrade.md

```diff
@@ -79,7 +79,7 @@ Then apply the latest changes from v1.11 and update the Rook Operator image.
 
 ```console
 kubectl replace -f common.yaml -f crds.yaml
```

Comment:
Likely conflicting with the `kubectl apply...` change just merged

Reply:
Yes that was it, thanks. fixed now